Dockerfile

```diff
@@ -1,8 +1,48 @@
-FROM clouder/base:3.4
-MAINTAINER Ted Salmon tsalmon@laslabs.com
+FROM alpine:latest
+MAINTAINER LasLabs Inc <support@laslabs.com>
 
-RUN apk add --no-cache samba-dc
 
-# Remove default Samba configs
-RUN rm -f /etc/samba/smb.conf
-RUN rm -rf /var/lib/samba/*
+ENV SAMBA_DC_ETC=/etc/samba
+ENV SAMBA_DC_DATA=/var/lib/samba
+ENV SAMBA_DC_LOGS=/var/log/samba
+ENV SAMBA_DC_REALM='corp.example.net'
+ENV SAMBA_DC_DOMAIN='EXAMPLE'
+ENV SAMBA_DC_ADMIN_PASSWD='5u3r53cur3!'
+ENV SAMBA_DC_DNS_BACKEND=SAMBA_INTERNAL
```

Comment:
Hmmm actually maybe it's only on filesystem operations. I'm digging around on Microbadger and it looks like `RUN` and `COPY` and the things generating layers. Can prolly disregard this then.

Might be better to use `.env` file though, which [turns out to be a thing](https://docs.docker.com/compose/env-file/)